src/main/java/org/owasp/esapi/logging/java/ESAPICustomJavaLevel.java

```diff
@@ -20,15 +20,15 @@
 /**
  *  A custom logging level defined between Level.SEVERE and Level.WARNING in logger.
  */
-public class ESAPIErrorJavaLevel extends Level {
+public class ESAPICustomJavaLevel extends Level {
```

Comment:
I meant to comment on this before. It's the main reason why I haven't accepted this PR. While I agree with the intent and this name does better reflect what this class is about, I think it is best **NOT** to change this class name because it could _potentially_ break someone's code that uses ESAPI. While I think it is _highly_ unlikely, someone could have been directly using `ESAPErrorJavaLevel`. (After all, it is a _public_ class library that does not hide components via Java modules and there is no "Do not use" warning in the class' Javadoc.) So, if we were to remove this, I'd have to call special attention to it in the next release notes (assuming that I'd remember).

If you really want to get rid of this class, then mark `ESAPIErrorJavaLevel` as deprecated, note in the Javadoc that it's being replaced by `ESAPICustomJavaLevel` and then we will mark it for removal in either 2 years after the next release or in the next major release (presumably 3.0, whenever that is). Then you can either copy/paste `ESAPIErrorJavaLevel` to `ESAPICustomJavaLevel` and tweak as needed (which is awkward, but probably best) or use delegation to that class. (I'm thinking that if you deprecate `ESAPIErrorJavaLevel` and then extended it as in:

`public class ESAPICustomJavaLevel extends ESAPIErrorJavaLevel`

the compiler probably would complain, at least if someone used `ESAPICustomJavaLevel`.)

However, in my opinion, I think the best thing to do is to just level the old class and wait until 3.0 to rename it. We'll have a whole new package namespace then (I'm thinking "org.owasp.esapi3") so we can then have much freer range of how to name things.

Reply:
@jeremiahjstacey -- If you don't think you can get to this for a while, I could just merge this PR and then do a follow-up PR to immediately flip the class names back. Let me know.

Reply:
I have restored the ESAPIErrorJavaLevel class from history and marked it as deprecated.
The documentation now suggests using the ESAPICustomJavaLevel.ERROR_LEVEL instead.

To ensure consistency between the ESAPIErrorJavaLevel and the ESAPICustomJavaLevel.ERROR_Level, I have used the values of ESAPICustomJavaLevel.ERROR_LEVEL as updated constructor args for the ESAPIErrorJavaLevel instantiation.